pages/events/index.vue

```diff
@@ -181,6 +185,10 @@ export default class extends QiskitPage {
       ? commit('addFilter', payload)
       : commit('removeFilter', payload)
   }
+
+  selectTab (selectedTab: boolean) {
```

Comment:
According to the [Carbon Vue story book](http://vue.carbondesignsystem.com/?path=/story/components-cvtabs--default), it seems the method's parameter is a number representing the tab index. Is there a reason for using `boolean` instead of `number`?